### PR TITLE
Migrate Profiles from ceramic to use dynamoDB: Part 2 - Update user

### PR DESCRIPTION
**Pull request by nagmak**
### What changes were made?

- Added 3 endpoints to update user in the backend: experience, company and profile
    - 3 endpoint functions: `updateUserProfile`, `updateUserExpertise`, `updateUserCompanyData`
- Allows users to add or edit the three areas of their profile

### Is there any background info about the change?

- As part of migration from ceramic to dynamoDB

### Are there any state changes? Mention key ones (if any)

```
setCompanyData({
      ...companyData,
      [evt.target.name]: evt.target.value
    });
```

### Screenshots or Gifs✨

Functional and Industry expertise are updating now:

Screen Shot 2022-03-18 at 3 54 49 PM
Screen Shot 2022-03-18 at 3 54 55 PM

Company Data is also updating now:

Screen Shot 2022-03-18 at 4 01 28 PM
Screen Shot 2022-03-18 at 4 01 44 PM

### Shortcut Link

1. https://app.shortcut.com/twali/story/335/migrate-profiles-from-ceramic-to-use-dynamodb
2. https://app.shortcut.com/twali/story/483/update-the-backend-to-match-functional-and-industry-expertise-arrays

### CODE REVIEW

**Comment by nagmak** [comment]
@NickGonzalez04 @hone1er I've updated with all the new changes! pls review and lmk if there's anything to fix :)

File: data/dynamodb.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
 const { v4 } = require("uuid");
 const TableName = process.env.TABLE_NAME;
+const AWS = require("aws-sdk");
```
Comment (NickGonzalez04) [inline data/dynamodb.js:3]: Was this not working inside the function?

File: data/dynamodb.js [hunk 2]
```diff
@@ -167,18 +167,121 @@ module.exports = {
...
+      const getParams = (updatedCompanyData) => {
+        const params = {
+          TableName,
+          Key: {
+            userWallet: userWallet,
+            userName: userName,
+          },
+          UpdateExpression: 'SET companyInfo = :updatedData',
+           ExpressionAttributeValues: {
+                ":updatedData": updatedCompanyData,
```
Comment (NickGonzalez04) [inline data/dynamodb.js:264]: LFGGGGGGGG 🚀

**Review by NickGonzalez04 — CHANGES_REQUESTED** [review]
PR is GREAT!! One ask, could you change the TableName back to `dev_staging_user_profile_test` ?

File: data/dynamodb.js (hunk 1 above)
Comment (nagmak) [reply data/dynamodb.js:3]: ah this is a remnant from the marshalling stuff, i'll move it back

**Comment by nagmak** [comment]
@NickGonzalez04 done!

**Review by NickGonzalez04 — APPROVED** [review]
MF merging this on Friday! Riskkkkkkkyyyy

**Pull request merged**